Sources/Sentry/SentryUIViewControllerSwizzling.m

```diff
@@ -354,7 +354,7 @@ - (void)swizzleLoadView:(Class)class
     // workaround, we skip swizzling the loadView and accept that the SKD doesn't create a span for
     // loadView if the UIViewController doesn't implement it.
     SEL selector = NSSelectorFromString(@"loadView");
-    IMP viewControllerImp = class_getMethodImplementation([UIViewController class], selector);
+    IMP viewControllerImp = class_getMethodImplementation([class superclass], selector);
```

Comment:
`m`: A short comment explaining this would be valuable cause I don't understand.